src/DependencyInjection/Configuration.php

```diff
@@ -652,6 +652,7 @@ private function getOrmEntityManagersNode(): ArrayNodeDefinition
                     ->scalarNode('class_metadata_factory_name')->defaultValue(ClassMetadataFactory::class)->end()
                     ->scalarNode('default_repository_class')->defaultValue(EntityRepository::class)->end()
                     ->scalarNode('auto_mapping')->defaultFalse()->end()
+                    ->booleanNode('enable_native_lazy_objects')->defaultFalse()->end()
```

Comment:
Please add a call to `info` describing what is happening when this is set to `false`.

Reply:
BTW, this node should probably not exist on PHP 8.4 :thinking: